challenges-webapp/src/main/webapp/vue-app/challenges/components/ChallengesList.vue

```diff
@@ -10,7 +10,10 @@
           md="6"
           lg="4"
           xl="3">
-          <challenge-card :challenge="challenge" @edit="$emit('edit-challenge',challenge)" />
+          <challenge-card
+            :challenge="challenge"
+            @edit="$emit('edit-challenge',challenge)"
+            @create-announce="$emit('create-announce',challenge)" />
```

Comment:
this is not needed anymore, isn't it, we may need to open the drawer from the card itself using `this.$root.$emit('open-announcement-drawer', challenge)` inside `<challenge-card`